packages/payment-integration-api/src/index.ts

```diff
@@ -147,6 +147,8 @@ export {
     PaypalInstrument,
     FormattedPayload,
     isHostedVaultedInstrument,
+    UntrustedShippingCardVerificationType,
+    PayPalInstrument,
```

Comment:
What is a difference between `PayPalInstrument` (line 151) and `PaypalInstrument` (line 147)?

Reply:
There's no difference. It's a redundant export. Thank you for your attention